pandas-stubs/core/frame.pyi

```diff
@@ -497,8 +497,7 @@ class DataFrame(NDFrame, OpsMixin):
     @overload
     def __getitem__(
         self,
-        idx: tuple
-        | Series[_bool]
+        idx: Series[_bool]
```

Comment:
`x = pd.DataFrame([[1, 2, 3], [4, 5, 6], [7, 8, 9]], index=["a", "b", "c"])`

```   
   0  1  2
a  1  2  3
b  4  5  6
c  7  8  9
```

`x[[0, 1]]` works

`x[(0, 1)]` does not, because that is interpreted as a column label